doc/Makefile.am

```diff
@@ -65,6 +65,7 @@ man-page-start: man-page-prepare
 
 man-page-build: upg-page-build man-page-start
 	@MAN_FILES=`find $(top_srcdir)/man/ -type f -name "coap.txt.in" ; find $(top_srcdir)/man/ -type f -name "coap_*.in" | ALL=C sort ; find $(top_srcdir)/man/ -type f -name "coap-*.in" | LC_ALL=C sort` ;\
+	MAN3_FILES=`find $(top_srcdir)/man/ -type f -name "coap_*.in" | ALL=C sort | LC_ALL=C sort` ;\
```

Comment:
The part `ALL=C sort` looks bogus.

Reply:
Correct.  All ALL=C corrected or removed as appropriate..